lib/src/detectors.js

```diff
@@ -200,6 +200,63 @@ const addDetectorApi = (matroid) => {
       this._genericRequest(options, resolve, reject);
     });
   };
+
+  matroid.addFeedbackFromFile = function(detectorId, imageFile, feedback) {
+    /*
+     * Add feedback to a Matroid detector from a local file.
+     */
+    return new Promise((resolve, reject) => {
+      const feedbackToAdd = Array.isArray(feedback) ? feedback : [feedback]
+      
+      const options = {
+        action: 'addFeedbackFromFile',
+        uriParams: { ':detectorId': detectorId },
+        filePaths: imageFile,
+        data: {
+          feedback: feedbackToAdd.map(item => JSON.stringify(item)),
+        }
+      };
+
+      this._genericRequest(options, resolve, reject);
+    });
+  }
+
+  matroid.addFeedbackFromURL = function(detectorId, url, feedback) {
+    /*
+     * Add feedback to a Matroid detector a url
+     */
+    return new Promise((resolve, reject) => {
+      const options = {
+        action: 'addFeedbackFromURL',
+        uriParams: { ':detectorId': detectorId },
+        data: {
+          feedback: Array.isArray(feedback) ? feedback : [feedback],
+          url,
+        }
+      };
+
+      this._genericRequest(options, resolve, reject);
+    });
+  }
```

Comment:
It could be good to check the arguments for `addFeedbackFromFile` and `addFeedbackFromURL` using `this._checkRequiredParams`